code/datums/objective/rev.dm

```diff
@@ -33,3 +33,29 @@
 		return TRUE
 
 	return FALSE
+
+/datum/objective/timed/excelsior
+	var/detect_timer = 60 MINUTES
+	var/active = FALSE
+	var/convert_decrease = 8 MINUTES
+	var/mandate_increase = 12 MINUTES
+
+/datum/objective/timed/excelsior/New()
+	..()
+	explanation_text = "Expand and grow in power before the ship's systems detect your presence! The detection countdown of [time2hours(detect_timer)] Hour starts once you force-implant a new comrade. It is lowered by [time2minutes(convert_decrease)] Minutes for each additional recruit, and increased by [time2minutes(mandate_increase)] Minutes for each completed mandate"
+
+/datum/objective/timed/excelsior/proc/start_excel_timer()
+	START_PROCESSING(SSobj, src)
+	active = TRUE
+
+/datum/objective/timed/excelsior/Process()
+	detect_timer -= 1 SECONDS
```

Comment:
Are you sure about this?
Every how many seconds the object subsystem is executed?
and if you know, where have you seen this information?

Reply:
https://github.com/discordia-space/CEV-Eris/blob/5151115192f3bcad5734153d049e5100d32a24e5/code/datums/objective/merc_mission.dm#L43 I just copied this.

Reply:
According to my calculations, each tick equals 1.1265 seconds. So the timer you have set will take longer than expected to reach the limit. 

Approximately 67.59 instead of the expected 60 seconds.

Reply:
Thank you, will try to fix this.